src/lib/pages/schedule-view.svelte

```diff
@@ -393,6 +394,7 @@
             policies={schedule?.schedule?.policies}
             notes={schedule?.schedule?.state?.notes}
           />
+          <ScheduleSearchAttributes schedule={schedule?.schedule} />
```

Comment:
Do we want to list search attributes in the schedule view? Or should it just be on create/edit?